BUILD.bazel

```diff
@@ -2,12 +2,6 @@ load("@bazel_skylib//rules:common_settings.bzl", "bool_flag")
 load("//build/nfpm:rules.bzl", "nfpm_pkg")
 load("@bazel_skylib//lib:selects.bzl", "selects")
 
-filegroup(
-    name = "srcs",
-    srcs = glob(["**"]),
```

Comment:
not including all files (even it's not from kong), as they are not needed during the build, is helpful when reducing bazel ram